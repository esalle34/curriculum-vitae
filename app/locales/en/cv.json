{
  "title": "Curriculum Vitae - Eric Salle",
  "profil": "Profile",
  "description": "<0>As frontend software engineer since 2017</0>, i bring new nice user interface features and do the maintenance of your websites and apps.",
  "qualities": "<0>Curious and always seeking new opportunities to learn</0>, i like to communicate and bring new skills to my team members.",
  "remote": "Furthermore, having experience as remote software engineer, i am able to work with appreciable <0>autonomy</0> as a remote team member.",
  "altaprofitsTitle": "Altaprofits <0>March 2017- March 2020</0>",
  "altaprofitsIntroduction": "Within a team of 3 people, maintenance and development of front-end interfaces.",
  "altaprofitsWork": [
    "Setting up the React library",
    "New frontend features",
    "New back-office features",
    "Back office management",
    "Site maintenance"
  ],
  "wefightTitle": "Wefight <0>April 2021 - October 2022</0>",
  "wefightIntroduction": "With the front-end chapter made up of 5 people, design of interfaces for the back office as well as a patient monitoring platform.",
  "wefightWork": [
    "Design and integration of frontend interfaces in React - Redux",
    "Development on a Backend for frontend in Express",
    "Management of auth0 rights",
    "Implementation from a mongoDB database",
    "Typescript",
    "Unit tests, integration tests"
  ],
  "24sTitle": "24s (LVMH group) <0>October 2022 - ...</0>",
  "24sIntroduction": "Within the web chapter made up of 4 people, maintenance and development of new features for the 24s e-commerce platform.",
  "24sWork": [
    "Frontend interface design in React - Redux",
    "Development on a Next base",
    "Intervention within several feature teams",
    "Integration of design from figma models",
    "Typescript",
    "Unit tests, integration tests"
  ],
  "hardSkills": "Skills",
  "languages": "Languages",
  "french": "French",
  "english": "English",
  "spanish": "Spanish",
  "contact": "Contact me"
}
