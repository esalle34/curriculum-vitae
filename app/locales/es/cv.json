{
  "title": "Curriculum Vitae - Eric Salle",
  "profil": "Perfil",
  "description": "<0>Desarrollador front-end desde 2017</0>, creo funcionalidades y mantengo tus sitios y aplicaciones.",
  "qualities": "<0>Curioso y siempre buscando novedades</0>, disfruto comunicándome e introduciendo a mis colegas nuevos métodos y nuevas herramientas.",
  "remote": "Por fin, gracias a mi experiencia remota, puedo responder a sus solicitudes a distancia <0>con total autonomía</0>.",
  "altaprofitsTitle": "Altaprofits <0>Marzo 2017- Marzo 2020</0>",
  "altaprofitsIntroduction": "Dentro de un equipo de 3 personas, mantenimiento y desarrollo de interfaces front-end.",
  "altaprofitsWork": [
    "Configurando la biblioteca React",
    "Nuevas funcionalidades de la interfaz",
    "Nuevas funcionalidades del back-office",
    "Gestión de back office",
    "Mantenimiento del sitio"
  ],
  "wefightTitle": "Wefight <0>Abril 2021 - Octubre 2022</0>",
  "wefightIntroduction": "Con el capítulo front-end conformado por 5 personas, diseño de interfaces para el back office así como una plataforma de seguimiento de pacientes.",
  "wefightWork": [
    "Diseño e integración de interfaces frontend en React - Redux",
    "Desarrollo en un Backend para frontend en Express",
    "Gestión de derechos auth0",
    "Implementación desde una base de datos mongoDB",
    "Typescript",
    "Pruebas de integración, pruebas de unidad"
  ],
  "24sTitle": "24s (grupo LVMH) <0>Octubre 2022 - ...</0>",
  "24sIntroduction": "Dentro del capítulo web conformado por 4 personas, mantenimiento y desarrollo de nuevas funcionalidades para la plataforma de comercio electrónico 24s.",
  "24sWork": [
    "Diseño de interfaz frontend en React - Redux",
    "Desarrollo sobre una base Next",
    "Intervención en varios equipos",
    "Integración del diseño de los modelos figma",
    "Typescript",
    "Pruebas de integración, pruebas de unidad"
  ],
  "hardSkills": "Habilidades",
  "languages": "Idiomas",
  "french": "Francés",
  "english": "Inglés",
  "spanish": "Español",
  "contact": "Contactar"
}
