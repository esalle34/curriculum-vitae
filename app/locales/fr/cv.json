{
  "title": "Curriculum Vitae - Eric Salle",
  "profil": "Profil",
  "description": "<0>Développeur front-end depuis 2017</0>, je crée des fonctionnalités et réalise la maintenance de vos sites et applications.",
  "qualities": "<0>Curieux et toujours à la recherche de la nouveauté</0>, j'apprécie communiquer et faire découvrir à mes collègues de nouvelles méthodes et de nouveaux outils.",
  "remote": "Enfin, fort d'une expérience en remote je suis capable de pourvoir à vos demandes à distance <0>en toute autonomie</0>.",
  "altaprofitsTitle": "Altaprofits <0>Mars 2017- Mars 2020</0>",
  "altaprofitsIntroduction": "Au sein d'une équipe de 3 personnes, maintenance et développement des interfaces front-end.",
  "altaprofitsWork": [
    "Mise en place de la librairie React",
    "Nouvelles fonctionnalités frontend",
    "Nouvelles fonctionnalités back-office",
    "Gestion du back-office",
    "Maintenance du site"
  ],
  "wefightTitle": "Wefight <0>Avril 2021 - Octobre 2022</0>",
  "wefightIntroduction": "Avec le chapitre front-end constitué de 5 personnes, conception d'interfaces à destination du back-office ainsi que d'une plateforme de suivi patient.",
  "wefightWork": [
    "Conception et intégration d'interfaces frontend en React - Redux",
    "Développement sur un Backend for frontend en Express",
    "Gestion des droits auth0",
    "Mise en place à partir d'une base mongoDB",
    "Typescript",
    "Tests unitaire, tests d'intégration"
  ],
  "24sTitle": "24s (LVMH group) <0>Octobre 2022 - ...</0>",
  "24sIntroduction": "Au sein du chapitre web constitué de 4 personnes, maintenance et développement de nouvelles fonctionnalités pour la plateforme e-commerce de 24s.",
  "24sWork": [
    "Conception d'interfaces frontend en React - Redux",
    "Développement sur un socle en Next",
    "Intervention au sein de plusieurs feature team",
    "Integration du design à partir de maquettes figma",
    "Typescript",
    "Tests unitaire, tests d'intégration"
  ],
  "hardSkills": "Compétences",
  "languages": "Langues",
  "french": "Français",
  "english": "Anglais",
  "spanish": "Espagnol",
  "contact": "Contact"
}
