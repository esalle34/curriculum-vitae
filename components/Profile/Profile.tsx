import ProfileImage from "../ProfileImage";
import ProfileSkills from "../ProfileSkills";
import { Props as ProfileSkillsProps } from "../ProfileSkills/ProfileSkills";
import ProfileSocialeMedia from "../ProfileSocialMedia";
import { Props as ProfileSocialMedia } from "../ProfileSocialMedia/ProfileSocialMedia";

interface Props {
  profileSkills: ProfileSkillsProps;
  profileSocialMedial: ProfileSocialMedia;
}

const Profile = ({ profileSkills, profileSocialMedial }: Props) => (
  <div className="md:ml-4 flex flex-col w-full h-max md:w-60  md:border-2 rounded border-blue-800 md:shadow-xl">
    <ProfileImage />
    <ProfileSkills
      hardSkills={profileSkills.hardSkills}
      languages={profileSkills.languages}
      english={profileSkills.english}
      french={profileSkills.french}
      spanish={profileSkills.spanish}
    />
    <ProfileSocialeMedia contact={profileSocialMedial.contact} />
  </div>
);
export default Profile;
