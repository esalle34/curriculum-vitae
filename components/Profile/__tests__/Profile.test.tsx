import { render } from "@testing-library/react";
import Profile from "../Profile";
import * as dict from "@/app/locales/fr/cv.json";

describe("Profile", () => {
  it("should match snapshot correctly", () => {
    const { container } = render(
      <Profile
        profileSkills={{
          hardSkills: dict.hardSkills,
          languages: dict.languages,
          french: dict.french,
          english: dict.english,
          spanish: dict.spanish,
        }}
        profileSocialMedial={{
          contact: dict.contact,
        }}
      />
    );
    expect(container).toMatchSnapshot();
  });
});
