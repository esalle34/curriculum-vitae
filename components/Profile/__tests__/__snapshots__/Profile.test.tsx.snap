// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile should match snapshot correctly 1`] = `
<div>
  <div
    class="md:ml-4 flex flex-col w-full h-max md:w-60  md:border-2 rounded border-blue-800 md:shadow-xl"
  >
    <img
      class="h-48 md:w-60 md:h-60"
      src="/images/Diabolo.jpeg"
    />
    <div
      class="bg-blue-800 border-blue-800 border-t-2 rounded-t md:rounded-none text-white p-3"
    >
      <p
        class="text-xl pb-4 text-center font-semibold"
      >
        Compétences
      </p>
      <ul
        class="text-sm"
      >
        <li
          class="flex justify-between"
        >
          Javascript
          <span
            class="flex"
          >
            <img
              src="/images/yellowStar.png"
              width="20"
            />
            <img
              src="/images/yellowStar.png"
              width="20"
            />
            <img
              src="/images/yellowStar.png"
              width="20"
            />
            <img
              src="/images/yellowStar.png"
              width="20"
            />
          </span>
        </li>
        <li
          class="flex justify-between"
        >
          CSS
          <span
            class="flex"
          >
            <img
              src="/images/yellowStar.png"
              width="20"
            />
            <img
              src="/images/yellowStar.png"
              width="20"
            />
            <img
              src="/images/yellowStar.png"
              width="20"
            />
          </span>
        </li>
        <li
          class="flex justify-between"
        >
          React
          <span
            class="flex"
          >
            <img
              src="/images/yellowStar.png"
              width="20"
            />
            <img
              src="/images/yellowStar.png"
              width="20"
            />
            <img
              src="/images/yellowStar.png"
              width="20"
            />
            <img
              src="/images/yellowStar.png"
              width="20"
            />
          </span>
        </li>
        <li
          class="flex justify-between"
        >
          Typescript
          <span
            class="flex"
          >
            <img
              src="/images/yellowStar.png"
              width="20"
            />
            <img
              src="/images/yellowStar.png"
              width="20"
            />
            <img
              src="/images/yellowStar.png"
              width="20"
            />
          </span>
        </li>
        <li
          class="flex justify-between"
        >
          Next
          <span
            class="flex"
          >
            <img
              src="/images/yellowStar.png"
              width="20"
            />
            <img
              src="/images/yellowStar.png"
              width="20"
            />
            <img
              src="/images/yellowStar.png"
              width="20"
            />
          </span>
        </li>
      </ul>
      <p
        class="text-xl py-4 text-center font-semibold"
      >
        Langues
      </p>
      <ul>
        <li
          class="flex justify-between"
        >
          Français
          <span
            class="flex"
          >
            <img
              src="/images/yellowStar.png"
              width="20"
            />
            <img
              src="/images/yellowStar.png"
              width="20"
            />
            <img
              src="/images/yellowStar.png"
              width="20"
            />
            <img
              src="/images/yellowStar.png"
              width="20"
            />
            <img
              src="/images/yellowStar.png"
              width="20"
            />
          </span>
        </li>
        <li
          class="flex justify-between"
        >
          Anglais
          <span
            class="flex"
          >
            <img
              src="/images/yellowStar.png"
              width="20"
            />
            <img
              src="/images/yellowStar.png"
              width="20"
            />
            <img
              src="/images/yellowStar.png"
              width="20"
            />
          </span>
        </li>
        <li
          class="flex justify-between"
        >
          Espagnol
          <span
            class="flex"
          >
            <img
              src="/images/yellowStar.png"
              width="20"
            />
            <img
              src="/images/yellowStar.png"
              width="20"
            />
            <img
              src="/images/yellowStar.png"
              width="20"
            />
            <img
              src="/images/yellowStar.png"
              width="20"
            />
          </span>
        </li>
      </ul>
    </div>
    <div
      class="bg-blue-800 border-blue-800 text-white border-b-2 md:border-0 rounded-b md:rounded-none p-3"
    >
      <p
        class="text-xl pb-4 text-center font-semibold"
      >
        Contact
      </p>
      <div
        class="text-sm flex flex-col gap-4"
      >
        <a
          class="flex justify-between items-center"
          href="https://www.linkedin.com/in/%F0%9F%92%BB-eric-salle-431664109/"
        >
          <img
            src="/images/LinkedIn_icon.svg"
            width="30"
          />
           Eric Salle
        </a>
        <a
          class="flex justify-between items-center"
          href="tel:+33683790289"
        >
          <img
            src="/images/smartphone.svg"
            width="30"
          />
           +33 6 83 79 02 89
        </a>
        <a
          class="flex justify-between items-center"
          href="mail:esalle34@gmail.com"
        >
          <img
            src="/images/mail.svg"
            width="30"
          />
           esalle34@gmail.com
        </a>
      </div>
    </div>
  </div>
</div>
`;
